src/settings.ts

```diff
@@ -594,3 +594,5 @@ export const SCHEDULE_SHOW_DAYS_AHEAD = 7;
  * @see https://firebase.google.com/docs/firestore/query-data/queries#in_not-in_and_array-contains-any
  */
 export const FIRESTORE_QUERY_IN_ARRAY_MAX_ITEMS = 10;
+
+export const REMOVE_EVENTS_FROM_VENUE = /^poster[0-9]{4}$/; // or "INCLUDE_ALL"
```

Comment:
I've now removed that — an empty string would be sufficient to not filter any venues if need be.

Reply:
This is the currently unresolved, and any suggestion regarding how to properly address this would be helpful. Filtering based on template type won't work for the use case here, but I think this string search is sufficiently specific 'poster[0-9][0-9][0-9][0-9]' to not impact other venues that are not posters. This is a very OHBM specific feature though, so happy to merge directly to env/ohbm if that would be preferable.